torch/testing/_internal/common_nn.py

```diff
@@ -1248,6 +1248,13 @@ def fractional_max_pool3d_test(test_case):
             fullname='FractionalMaxPool3d_asymsize')
 
 
+def single_batch_reference_fn(ref_input, parameters, ref_module):
+    """Reference function for module supporting no batch dimensions."""
```

Comment:
I like the idea of this function! It could be a little bit clearer that the purpose of this is to run an input through in batched form with a single item (for comparison against true no-batch inputs)